lib/jekyll-import/importers/roller.rb

```diff
@@ -97,28 +97,31 @@ def self.process(opts)
                            :encoding => "utf8")
 
         posts_query = "
-           SELECT
-             weblogentry.id           AS `id`,
-             weblogentry.status       AS `status`,
-             weblogentry.title        AS `title`,
-             weblogentry.link         AS `slug`,
-             weblogentry.updatetime   AS `date`,
-             weblogentry.text         AS `content`,
-             weblogentry.summary      AS `excerpt`,
-             weblogentry.categoryid   AS `categoryid`,
-             roller_user.fullname     AS `author`,
-             roller_user.username     AS `author_login`,
-             roller_user.emailaddress AS `author_email`
-           FROM weblogentry AS `weblogentry`
-             LEFT JOIN roller_user AS `roller_user`
-               ON weblogentry.creator = roller_user.username"
+                 SELECT
```

Comment:
You can ask, but I'm afraid the answer is probably not very satisfying.

This is the string I was having trouble being frozen, and in my naivety, I assumed the issue lay somewhere around here, so I made many changes to this string. I had thought I had put everything back as it was, but clearly not. After my changes and running `bundle exec rubocop --auto-correct` again this is what the result is. I'd love to blame rubocop, but it is far more likely my ad-hoc corrections were to blame.

Simply put it didn't, and this got included in my commit because of how I was editing the file. The fix is the two changes of `<<` to `+=` further down.

This string was changed with the addition of three lines to get the extra information I used to set the permalink.

Reply:
Thank you for the response @pmb00cs 
I now see that you had added the following to lines to the "query string" as well, which was not immediately evident from the diff.
```
  LEFT JOIN weblog AS `weblog`
    ON weblogentry.websiteid = weblog.id
```

There is a bit of an inconsistency regarding the "query strings" in the class overall, though..
Here, the string starts with a `\n`..
```ruby
  posts_query = "
        SELECT
          weblogentry.id           AS `id`,
          ..."
```
elsewhere there is this:
```
  cquery =
    "SELECT
       weblogcategory.name AS `name`
```

Unfortunately, since I don't know how these strings are ultimately handled (no tests!), I won't be able to suggest a refactoring..